### ISSUE TITLE
Option to change default search engine when more is pressed 

### ISSUE THREAD

**Issue by the-wat** [meetDeveloper/Dictionary-Anywhere#56]
![image](https://user-images.githubusercontent.com/69153476/136751745-88fc9a5e-4fcb-4832-aaa8-db156b531a93.png)

Upon pressing more, it redirects to a Google search, so basically allow a larger customization by creating a toggle for search engine used. Some people prefer DDG and other search engines, so they might prefer to use the same for word definition searching.

**Comment by onlycparra** [comment]
Yes, please. Not all of us want to use the same source for our definitions.

**Comment by akvedi** [comment]
One this issue [here](https://github.com/akvedi/lexicon), I will add it soon.
